src/Api/Vault/Controllers/SyncController.cs

```diff
@@ -71,7 +71,8 @@ public async Task<SyncResponseModel> Get([FromQuery] bool excludeDomains = false
                 ProviderUserStatusType.Confirmed);
         var hasEnabledOrgs = organizationUserDetails.Any(o => o.Enabled);
         var folders = await _folderRepository.GetManyByUserIdAsync(user.Id);
-        var ciphers = await _cipherRepository.GetManyByUserIdAsync(user.Id, hasEnabledOrgs);
+        var allCiphers = await _cipherRepository.GetManyByUserIdAsync(user.Id, hasEnabledOrgs);
```

Comment:
suggestion: I don't think this change does anything and can likely be reverted.

Reply:
Reverted this in [4486510](https://github.com/bitwarden/server/pull/2679/commits/4486510538ce44227cf56776e560eb127f83fc20).  It looks like it was left over from a version check that Kyle put in that has since been refactored out.